### PR TITLE
PR-3422

### PR DESCRIPTION
**Pull request by yasuflatland-lf**

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by liuxingye** [comment]
ci:test:relevant

**Comment by liuxingye** [comment]
ci:test:sf

File: modules/apps/translation/translation-translator-deepl/src/main/java/com/liferay/translation/translator/deepl/internal/translator/DeepLClient.java [hunk 1]
```diff
@@ -42,6 +41,16 @@
...
+	public static final String AUTH_KEY = "auth_key";
+
+	public static final String SOURCE_LANG = "source_lang";
+
+	public static final String TARGET = "target";
+
+	public static final String TARGET_LANG = "target_lang";
+
+	public static final String TEXT = "text";
+
```
Comment (adolfopa) [inline modules/apps/translation/translation-translator-deepl/src/main/java/com/liferay/translation/translator/deepl/internal/translator/DeepLClient.java:53]: Didn't bchan ask us to inline them?
Anyway, if you don't need to inline them, you need to make these private and rename them to "_AUTH_KEY", etc.

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/translation/translation-translator-deepl/src/main/java/com/liferay/translation/translator/deepl/internal/translator/DeepLClient.java (hunk 1 above)
Comment (yasuflatland-lf) [reply modules/apps/translation/translation-translator-deepl/src/main/java/com/liferay/translation/translator/deepl/internal/translator/DeepLClient.java:53]: @adolfopa Does Inline mean hard corded? Then I'll do that , squash and send PR to liferay-lima again.

**Review by adolfopa — APPROVED** [review]

**Pull request closed without merging**